core/org.eclipse.cdt.core.native/src/org/eclipse/cdt/utils/pty/PTY2.java

```diff
@@ -0,0 +1,69 @@
+/********************************************************************************
+ * Copyright (c) 2022 徐持恒 Xu Chiheng
```

Comment:
Just to be explicit in case I am being unclear in the main message:

```suggestion
 * Copyright (c) 2015, 2022 徐持恒 Xu Chiheng
```

Please refer to https://www.eclipse.org/projects/handbook/#ip-copyright-headers for more detail